src/components/material/material-buttons/MaterialButtons.tsx

```diff
@@ -18,21 +18,28 @@ const MaterialButtons: FC<MaterialButtonsProps> = ({
   manifestation: { pid },
   size
 }) => {
-  const hasPhysicalAccess = manifestation.accessTypes.some(
-    (type) => type.code === AccessTypeCode.Physical
-  );
+  const hasCorrectAccessType = (
+    desiredAccessType: AccessTypeCode,
+    manifest: Manifestation
+  ) => {
+    return !!manifest.accessTypes.some(
+      (type) => type.code === desiredAccessType
+    );
+  };
```

Comment:
Consider whether this is something that is more widely usable.

Checking for an access type seems like something we would do a lot of places. If we have a good implementation for this then it should be in a helper and used where we can.

Reply:
I see your point, and even though we're currently not using it any other places, I think it could be a helpful function in the future (in case we ever need it).